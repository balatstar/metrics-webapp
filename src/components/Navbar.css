header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  background-color: #ec4c8a;
}

header h1 {
  text-align: center;
  font-size: 18px;
  font-weight: 300;
  text-transform: lowercase;
}

.nav-home {
  padding-right: 10px;
}
