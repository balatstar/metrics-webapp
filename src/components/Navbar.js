import React from 'react';
import { Link } from 'react-router-dom';
import './Navbar.css';
import previous from './images/previous.png';
import home from './images/home.png';

function Navbar() {
  const goBack = () => {
    window.history.back();
  };

  return (
    <header>
      <div className="nav-back"><a onClick={goBack}><img src={previous} alt="Go Back" /></a></div>
      <h1>UK Carbon Intensity Indicator</h1>
      <div className="nav-home"><Link to="/"><img src={home} alt="Home" /></Link></div>
    </header>
  );
}
export default Navbar;
