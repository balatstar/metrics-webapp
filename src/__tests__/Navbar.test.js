import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import '@testing-library/jest-dom/extend-expect';
import Navbar from '../components/Navbar';

describe('Navbar', () => {
  it('renders Navbar component', () => {
    const { container } = render(
      <Router>
        <Navbar />
      </Router>,
    );
    expect(container).toMatchSnapshot();
  });
});
