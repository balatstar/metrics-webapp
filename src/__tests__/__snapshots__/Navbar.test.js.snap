// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders Navbar component 1`] = `
<div>
  <header>
    <div
      class="nav-back"
    >
      <img
        src="previous.png"
      />
    </div>
    <h1>
      UK Carbon Intensity Indicator
    </h1>
    <div
      class="nav-home"
    >
      <a
        href="/"
      >
        <img
          src="home.png"
        />
      </a>
    </div>
  </header>
</div>
`;
